frontend/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "description": "",
+  "main": "index.js",
+  "devDependencies": {},
+  "scripts": {
+    "start": "echo hello from frontend"
```

Comment:
Just a placeholder `start` script for now to test running things in parallel. This can get swapped out when we have a real frontend.